src/base/bittorrent/session.cpp

```diff
@@ -2063,6 +2063,23 @@ bool Session::addTorrent_impl(CreateTorrentParams params, const MagnetUri &magne
             patchedFastresumeData.replace("6:pausedi0e", "6:pausedi1e");
             patchedFastresumeData.replace("12:auto_managedi0e", "12:auto_managedi1e");
 
+            // converting relative save_path to absolute
+            int start = patchedFastresumeData.indexOf("9:save_path");
+            if (start > -1) {
+                start += 11;
+                const int end = patchedFastresumeData.indexOf(":", start);
+                const int len = patchedFastresumeData.mid(start, end - start).toInt();
```

Comment:
Seems you can use "Utils::ByteArray::midView()" here and in similar case below.

Reply:
Changed QByteArray.mid() to Utils::ByteArray::midView() in https://github.com/qbittorrent/qBittorrent/pull/11785/commits/9082423c24b7cbdded61507f4cbb35a263bc9397